packages/crayons-core/src/components/country-phone/readme.md

```diff
@@ -0,0 +1,103 @@
+# fw-country-phone
+
```

Comment:
Kindly add necessary documentation. Please add sample for to use as web component and as React component.